src/app/workspace/organization/detail/payments-panel/payments-panel.component.ts

```diff
@@ -160,17 +161,21 @@ export class PaymentsPanelComponent implements OnInit, OnDestroy, AfterViewInit
 
     this.notificationService.confirm(message, title, [
       {text: okButtonText, bold: true},
-    ]);
+    ]);*/
 
     if (!isNullOrUndefined($event) && $event !== '') {
-      //(window as any)._gopay.checkout({gatewayUrl: $event, inline: true});
+      (window as any)._gopay.checkout({gatewayUrl: $event, inline: true});
     }
   }
 
   public ngAfterViewInit(): void {
     const script = this.document.createElement('script');
     script.type = 'text/javascript';
-    script.src = 'https://gw.sandbox.gopay.com/gp-gw/js/embed.js';
+    if (environment.production) {
+      script.src = 'https://gate.gopay.com/gp-gw/js/embed.js';
+    } else {
+      script.src = 'https://gw.sandbox.gopay.com/gp-gw/js/embed.js';
```

Comment:
What about defining this URLs as another environment property?

Reply:
If I only knew how :-( Can you help me?